apps/src/lib/kits/maker/commands.js

```diff
@@ -0,0 +1,27 @@
+/** @file Maker commands (invoked by Applab/Gamelab.executeCmd) */
+import {apiValidateType} from '@cdo/apps/javascriptMode';
+
+const commands = {
+
+  /**
+   * Execute some code every X milliseconds.  This is effectively setInterval()
+   * with a cleaner interface.
+   * @param {number} opts.ms How often to invoke the code in the loop,
+   *        in milliseconds.
+   * @param {function(function)} opts.callback Code to invoke in each loop
+   *        iteration. Gets passed an 'exit' function that will stop the loop.
+   */
+  timedLoop(opts) {
```

Comment:
I thought about that and looked into this.  `apiValidateType` actually caches information about the parameters it's previously validated on the `opts` object, I assume to speed things up - but it does seem like it's important to pass.  It's too bad, because I agree that this would be cleaner in ES6.